src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -34,7 +34,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         List<Car> cars = carService.getAll();
         req.setAttribute("drivers", drivers);
         req.setAttribute("cars", cars);
-        req.getRequestDispatcher("/WEB-INF/views/service/addDriverToCar.jsp").forward(req, resp);
+        req.getRequestDispatcher("/WEB-INF/views/service/carsDriversAdd.jsp").forward(req, resp);
```

Comment:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/cars/drivers/add.jsp").forward(req, resp);
```
same with all others